File: src/jianyou/lin/FileSelectionController.java
package jianyou.lin;

import javafx.scene.control.Button;
import javafx.scene.control.TextField;
import javafx.scene.control.TreeItem;
import javafx.scene.control.TreeView;

public class FileSelectionController {
    public Button readPathBtn;
    public TreeView fileTree;
    public TextField pathText;

    public void readFilePath() {
        String pathTextText = pathText.getText();
        if (pathTextText.isEmpty()) {
            pathTextText = System.getProperty("user.dir");
            System.out.println("path:" + pathTextText);
        }
        TreeItem value = new TreeItem(pathTextText);
        fileTree.setRoot(value);
    }

}

File: src/jianyou/lin/FileSelectionController.kt
package jianyou.lin

import javafx.fxml.FXML
import javafx.scene.control.Button
import javafx.scene.control.TextField
import javafx.scene.control.TreeItem
import javafx.scene.control.TreeView

class FileSelectionController {
    @FXML
    var readPathBtn: Button? = null
    @FXML
    var fileTree: TreeView<String>? = null
    @FXML
    var pathText: TextField? = null


    fun readFilePath() {
        var pathTextText = pathText!!.text
        if (pathTextText.isEmpty()) {
            pathTextText = System.getProperty("user.dir")
            println("path:$pathTextText")
        }
        val value = TreeItem(pathTextText)
        fileTree!!.root = value
    }

}
